wagtail/wagtailimages/templates/wagtailimages/images/confirm_delete.html

```diff
@@ -12,6 +13,12 @@
             {% image image max-800x600 %}
         </div>
         <div class="col6">
+            {% usage_count_enabled as uc_enabled %}
+            {% if uc_enabled %}
+                <div class="usagecount">
+                    <a href="{{ image.usage_url }}">{% blocktrans count useage_count=image.get_usage.count %}Used {{ useage_count }} time{% plural %}Used {{ useage_count }} times{% endblocktrans %}</a>
```

Comment:
Typo here: `useage` instead of `usage`.

Reply:
@BertrandBordage I'll gladly fix it, but then i'll also fix it in the original template, requiring an update of translation files.